feat(APP-3954): Update Tab component to not render the TabList when there is a single child (#407)
aragon/gov-ui-kit: CHANGELOG.md

## [Unreleased]

### Changed

> - Update `Tabs.List` component to only render if we have multiple tabs.

Reviewer: ```suggestion
- Update `Tabs.List` component to only render when used with multiple tabs
```
